pkgs/by-name/am/amp-cli/package.nix

```
@@ -72,7 +72,7 @@ buildNpmPackage rec {
 
   meta = {
     description = "Amp is an AI coding agent, in research preview from Sourcegraph. This is the CLI for Amp.";
```

```suggestion
    description = "CLI for Amp, an AI coding agent in research preview from Sourcegraph";
```
In particular, it may not end in a period.

---

https://github.com/NixOS/nixpkgs/pull/410063 sorts it out